scss/_tables.scss

```diff
@@ -187,3 +187,20 @@
     }
   }
 }
+
+
+// Fixed header
+//
+// Table header with the `.thead-fixed` class applied fix its cells on top
+// allowing only <tbody> to scroll.
+
+// stylelint-disable-next-line no-duplicate-selectors
+.table {
+  .thead-fixed {
+    th {
+      position: sticky;
+      top: $table-head-fixed-margin-top;
```

Comment:
This isn't a `margin-top`, it's `top`. Variable name should reflect that: `$table-head-sticky-top`.

Reply:
Make sense. Done.